sct-commons/src/test/java/org/lfenergy/compas/sct/commons/dto/DaTypeNameTest.java

```diff
@@ -122,4 +122,33 @@ void testFrom() {
         // then
         assertThat(da2).isEqualTo(da1);
     }
+
+    /**
+     * ListFcEnum = CF, DC, SG, SP, ST , SE
+     * Fc value should be in ListFcEnum to be considered as known
+     */
+    @Test
+    void isUpdatable_ShouldReturnTrue_WhenValImportTrue_And_FCKnown() {
+        // given : nothing
+        DaTypeName da1 = createDa("da1.bda1.bda2", TFCEnum.CF, true, Map.of(0L, "value"));
+        // when then
+        assertThat(da1.isUpdatable()).isTrue();
+    }
+
+     @Test
```

Comment:
I think you didn't formatted your code ?

Reply:
????

Reply:
well it's just that there are some spaces, but it's not a big deal